pkg/volume/gce_pd/gce_util.go

```diff
@@ -82,8 +82,8 @@ func (gceutil *GCEDiskUtil) CreateVolume(c *gcePersistentDiskProvisioner) (strin
 	name := volume.GenerateVolumeName(c.options.ClusterName, c.options.PVName, 63) // GCE PD name can have up to 63 characters
 	capacity := c.options.PVC.Spec.Resources.Requests[v1.ResourceName(v1.ResourceStorage)]
 	requestBytes := capacity.Value()
-	// GCE works with gigabytes, convert to GiB with rounding up
-	requestGB := volume.RoundUpSize(requestBytes, 1024*1024*1024)
+	// GCE PDs are allocated in chunks of GBs (not GiBs)
+	requestGB := volume.RoundUpSize(requestBytes, volume.GB)
 
```

Comment:
This is really tiny nit - but can we simply use `volume.RoundUpToGB` and remove that `reuestBytes` variable?

Reply:
@gnufied, done, Thanks.